filer/templates/admin/filer/folder/directory_table.html

```diff
@@ -156,7 +152,7 @@
         </tbody>
     </table>
 
-    <div class="filer-dropzone-info-message js-filer-dropzone js-filer-dropzone-info-message hidden" data-url="{% if folder.id %}{% url 'admin:filer-ajax_upload' folder_id=folder.id %}{% else %}{% url 'admin:filer-ajax_upload' %}{% endif %}" data-folder-name="{% if folder.name and not folder.name == 'root' %}{{ folder.name }}{% else %}unfiled files{% endif %}" data-max-uploader-connections="{{ uploader_connections }}" data-max-file-size="20">
+    <div class="filer-dropzone-info-message js-filer-dropzone js-filer-dropzone-info-message hidden" data-url="{% if folder.id %}{% url 'admin:filer-ajax_upload' folder_id=folder.id %}{% else %}{% url 'admin:filer-ajax_upload' %}{% endif %}" data-folder-name="{% if folder.name and not folder.name == 'root' %}{{ folder.name }}{% else %}Unsorted Uploads{% endif %}" data-max-uploader-connections="{{ uploader_connections }}" data-max-file-size="20">
```

Comment:
``{% if folder.name and not folder.name == 'root' %}{{ folder.name }}{% else %}Unsorted Uploads{% endif %}``
Is this is the label of the dropzone when someone drags a file directly onto the "Folder List" view, right?

hmm... so if the folder has a name and the name is ``'root'``, then display ``Unsorted Uploads``?

Basically, whenever the current folder is not a real folder, it should say ``Unsorted Uploads``, right?

I'd say this can be simplified as well. @lory87 let's discuss in a hangout :-)